src/util/Handlers.js

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) 2018-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(['okta', 'util/Logger', 'util/OAuth2Util'], function (Okta, Logger, Util) {
+
+  var Handlers = {};
+  var _ = Okta._;
+
+  Handlers.filterOAuthRedirectParams = function(options, config) {
+    // Override specific OAuth/OIDC values
+    var renderOptions = {
+      el: options.el,
```

Comment:
Today, our documentation and samples suggest passing `el` during the `renderEl` function, not via [`new OktaSignIn()`](https://github.com/okta/okta-signin-widget/#new-oktasigninconfig). To be considerate of developers who want to upgrade their previous implementations to use this method, they can simply remove the `renderEl` functionality and use `showSignInToGetTokens` instead. If we removed the `el` prop here, we'd have conflicting usability instructions.

I was thinking about this as follows:
 - the `el` param should be overridable (as `clientId` and `redirectUri` are)
 - if we support multiple Widgets on the page, a developer could render them in separate components without needed to reinitialize the Widget

Granted, this could/should be solved by updating our `README`, but I think we need to consider the use-case where developers want override abilities for `el`.

Reply:
Per #487 it looks like the intention it to move the `el` declaration to the widget constructor, and I think this is a good thing.  The goal of this work is to make `renderEl()` an advanced use case, as such we need to move the `el` declaration upward.

IMO the "multiple widgets" conversation is out of scope until we have direction on a real use case. Besides, if I need multiple widgets, is it not conceptually simpler to create multiple instances of the widget?